.github/pull_request_template.md

```diff
@@ -15,10 +15,7 @@ The [AP CSP Create Performance Task](https://apcentral.collegeboard.org/courses/
   Links to relevant external resources; ie, specification documents, Jira tickets, related PRs, Honeybadger errors, etc.
 -->
 
-<!--
-- spec: []()
```

Comment:
is the implication that the spec (if there is one) should be linked from the jira ticket rather than the PR?

Reply:
I don't think I'm seeing the Spec link filled out often, and I would generally expect to find it in the Jira ticket if not here.

But that wasn't an explicit intention. My motivation here was:
* Eliminate the mouse & keyboard activity to uncomment this section - simplify to "click + paste"
* Eliminate the unnecessary `[]()` boilerplate - if you want to make it pretty, go for it, but it's not necessary

We could add `- Spec: ` back to the list.